server/views/pages/bookedtoday/arrivals/searchforexisting/noExistingRecordsFound.njk

```diff
@@ -1,43 +1,42 @@
 {% extends "../../../../partials/layout.njk" %}
 {% from "../../../../components/prisoner-summary-detail.njk" import prisonerSummaryDetail %}
-
 {% set pageTitle = "Confirm arrival" %}
-
 {% block content %}
-    <main class="govuk-main-wrapper">
```

Comment:
this and the removed main elements below were duplicate main elements causing extra spacing